include/swift/SIL/SILRemarkStreamer.h

```diff
@@ -0,0 +1,101 @@
+//===--- SILRemarkStreamer.h - Interface for streaming remarks --*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+/// \file
+/// This file defines the interface used to stream SIL optimization diagnostics
+/// through LLVM's RemarkStreamer interface.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_SIL_SILREMARKSTREAMER_H
+#define SWIFT_SIL_SILREMARKSTREAMER_H
+
+#include "swift/SIL/OptimizationRemark.h"
+#include "llvm/Remarks/RemarkStreamer.h"
+
+namespace swift {
+
+class SILRemarkStreamer {
+  llvm::remarks::RemarkStreamer &RS;
+  // Source manager for resolving source locations.
+  const SourceManager &SM;
```

Comment:
Same here. The rest of SIL doesn't follow SILModule's style. Please use camelCase for fields, arguments, and variables.